.github/workflows/tests.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Parallel tests
         run: python -m pytest --cov nanover python-libraries -n auto -m 'not serial'
       - name: Serial tests
-        run: python -m pytest --cov nanover python-libraries -n auto -m 'serial'
+        run: python -m pytest --cov nanover python-libraries -n 1 -m 'serial'
```

Comment:
This is probably our issue: serial tests need to be run in a single process

Reply:
Damn copy-paste! It is odd it did not blow up earlier!